crates/fuel-core/src/schema/dap.rs

```diff
@@ -132,7 +132,7 @@ impl ConcreteStorage {
         let fee_params = params.fee_params();
 
         let ready_tx = checked_tx
-            .into_ready(GAS_PRICE, gas_costs, fee_params)
+            .into_ready(GAS_PRICE, gas_costs, fee_params, None)
```

Comment:
I didn't add a todo because in this case it seems None is the right value, but please check @AurelienFT